index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <p>The man who saved a billion lives</p>
```

Comment:
- [ ] Well done @lenkon, your code is looking clean regarding the requirement, but we have a few things to modify in the project styling. Please make your best to match as much as possible to the design. The header is not the same as the one in the original design, please fix that.
![1c](https://user-images.githubusercontent.com/7733353/207127697-c144eaa7-fcd9-4f0a-805e-4197b702dc60.jpg)
- [ ] Following the Html best practice, the h1 and p tags could be included in a header section.